packages/app-mobile/components/screens/ConfigScreen/plugins/PluginBox/ActionButton.tsx

```diff
@@ -0,0 +1,35 @@
+import * as React from 'react';
+import { useCallback } from 'react';
+import { ItemEvent, PluginItem } from '@joplin/lib/components/shared/config/plugins/types';
+import { Button, ButtonProps } from 'react-native-paper';
+
+export type PluginCallback = (event: ItemEvent)=> void;
+
+interface Props extends Omit<ButtonProps, 'item'|'onPress'|'children'> {
```

Comment:
`Omit` is necessary here because `onPress` has a different type in `ButtonProps` and we use `title` rather than `children` to set the content of the button.